include/genericls/UsgsAstroLsStateData.h

```diff
@@ -85,7 +85,9 @@ class UsgsAstroLsStateData
    int          m_PlatformFlag;                   // 7
    int          m_AberrFlag;                      // 8
    int          m_AtmRefFlag;                     // 9
-   double       m_IntTime;                        // 10
+   std::vector<double> m_IntTimeLines;
+   std::vector<double> m_IntTimeStartTimes;
+   std::vector<double> m_IntTimes;

```

Comment:
These could be changed to a vector of structs. This would make some things a bit safer and better match ISIS.